modules/nw-ingress-sharding-default.adoc

```diff
@@ -0,0 +1,55 @@
+// Module include in the following assemblies:
+//
+// * ingress-operator.adoc
+// * networking/ingress-sharding.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-default_{context}"]
+= Sharding the default Ingress Controller
+
+After creating a new Ingress shard, there could be routes that are admitted for your new Ingress shard that are also admitted by the default Ingress Controller. This is because the default Ingress Controller has no selectors and admits all routes by default.
+
+The following procedure restricts the default Ingress Controller from servicing your newly sharded `finance`, `ops`, and `dev`, routes. This adds further isolation to Ingress shards.
+
+[IMPORTANT]
+====
+You must keep all of {product-title}'s administration routes on the same Ingress Controller. Therefore, use the default Ingress Controller as outlined by the following procedure. Avoid adding additional selectors to the default Ingress Controller that exclude these essential routes.
```

Comment:
A bit of a nit pick, but the wording (that I think I wrote) sounded a bit too specifically demanding, and maybe a bit redundant. Users won't be able to exactly follow these instructions in `Sharding the default Ingress Controller`, so probably makes sense to be more generic and just tell them to avoid excluding these essential routes.

```suggestion
You must keep all of {product-title}'s administration routes on the same Ingress Controller. Therefore, avoid adding additional selectors to the default Ingress Controller that exclude these essential routes.
```